pyproject.toml

```
@@ -16,42 +16,15 @@ author = "napari"
 url = "https://napari.org/"
 license = "BSD license"
 # version populated in bundle.py
+version = "0.3.7rc11.dev7"
```

```suggestion
version = "0.0.0"
```
If I understand correctly and this gets overwritten in the bundle can we go for a more neutral option here? I can just imagine someone being confused and wondering what was special about those numbers

---

It will also maybe make things more clear if the overwrite fails to happen correctly

---

Yep, sorry meant to change that.  But let’s make it 0.0.1 ... since briefcase on windows will fail with 0.0.0 (if someone tried to build directly without using bundle.py)